src/class/cdc/cdc_device.h

```diff
@@ -133,6 +133,9 @@ static inline bool     tud_cdc_write_clear     (void);
 // Invoked when received new data
 TU_ATTR_WEAK void tud_cdc_rx_cb(uint8_t itf);
 
+// Invoked when received send break
+TU_ATTR_WEAK void tud_cdc_send_break_cb(uint8_t itf, uint16_t wait_ms);
```

Comment:
can you move it after `tud_cdc_line_coding_cb()`, also change the `wait_ms` to `duration_ms`.